packages/flutter_tools/lib/src/commands/daemon.dart

```diff
@@ -198,8 +215,147 @@ class AppDomain extends Domain {
   }
 }
 
+/// This domain lets callers list and monitor connected devices.
+///
+/// It exports a `getDevices()` call, as well as firing `device.added`,
+/// `device.removed`, and `device.changed` events.
+@domain
```

Comment:
The `@domain` and `@command` ones are used to demarcate the JSON-RPC domains and their commands, but I'm not using the info in the code - no programmatic reflection. I can strip them out.